core/modules/system/system.theme.inc

```diff
@@ -462,7 +462,7 @@ function theme_admin_block($variables) {
 
   $output .= '<div class="' . implode(' ', $classes) . '">';
   if (!empty($block['title'])) {
-    $output .= '<h3>' . $block['title'] . '</h3>';
+    $output .= '<h3 class="has-icon">' . $block['title'] . '</h3>';
```

Comment:
My understanding of front-end practices is that we should be giving this a class that can be used alone without the tag name, such as `admin-panel-heading` or `admin-panel-title`. Then when we have selectors, instead of `.admin-panel h3.has-icon`, we would use the single class `.admin-panel-title`. That makes it so that you have a low level of specificity for themes to override.

We already had 2 levels of specificity with `.admin-panel h3`, changing it to `.admin-panel h3.has icon` is now 3 levels.

Reply:
I was trying to keep it as close to 1.19.1 as possible, I'm worried that by changing the level of level of specificity in addition to adding a class is going to cause a lot of problems with admin themes.  

I like this approach for 2.x, but since we are already trying to fix a regression, I think we should change as little as possible.

I'm going to close this PR in favor of @olafgrabienski's, which keeps things closer to how they were in 1.19.1 before the visual break.